content/en/docs/concepts/workloads/pods/downward-api.md

```diff
@@ -75,8 +78,9 @@ via either mechanism are:
 `status.podIP`
 : the pod's primary IP address (usually, its IPv4 address)
 
-In addition, the following information is available through
-a `downwardAPI` volume `fieldRef`, but **not as environment variables**:
+The following information is available through a `downwardAPI` volume 
+`fieldRef`,
+**but not as environment variables**:
```

Comment:
Nit: This extra line wrapping is not needed.

Reply:
Addressed this change. PTAL!!